app/Metadata/Versions/FileVersion.php

```diff
@@ -44,7 +45,7 @@ public function __construct()
 	 */
 	public function hydrate(bool $withRemote = true, bool $useCache = true): void
 	{
-		if ($withRemote) {
+		if ($withRemote && Schema::hasTable('configs')) {
```

Comment:
Is that needed in case of an incomplete installation?

Reply:
yes, otherwise it fails with key not found in config.